planning/behavior_path_planner/autoware_behavior_path_goal_planner_module/src/goal_planner_module.cpp

```diff
@@ -1438,7 +1442,9 @@ BehaviorModuleOutput GoalPlannerModule::planPullOverAsOutput(
 
   // return to lane parking if it is possible
   if (is_freespace && canReturnToLaneParking(context_data_with_velocity)) {
-    thread_safe_data_.set_pull_over_path(thread_safe_data_.get_lane_parking_pull_over_path());
+    if (context_data_with_velocity.pull_over_path_opt) {
+      thread_safe_data_.set_pull_over_path(context_data_with_velocity.pull_over_path_opt.value());
+    }
```

Comment:
is it possible to return freespace path to laneparking path?
`context_data_with_velocity.pull_over_path_opt.value())` is also freespace path?

Reply:
I added comment